packages/ui-components/src/Input/CommonInput.tsx

```diff
@@ -41,15 +43,17 @@ export const CommonInput = ({
 
   return (
     <div
-      className={classNames("input-container", className, {
+      className={classNames(className, {
         inline: inline,
       })}
     >
       {fieldInput}
-      <div className="message">
-        {errorMsg}
-        {helpMsg}
-      </div>
+      {(errorMsg || helpMsg) && (
+        <div className="message">
+          {errorMsg}
+          {helpMsg}
+        </div>
+      )}
```

Comment:
to match the behaviour in Figma, only show `padding` under the `input` if there is a `help/error message`